wikidict/lang/de/__init__.py

```diff
@@ -0,0 +1,54 @@
+"""Deutsh language."""
+
+# Regex to find the pronunciation
+pronunciation = r"{{Lautschrift\|([^}]+)}}"
+
+# Regex to find the gender
+gender = r",\s+{{([fmn]+)}}"
+
+# Float number separator
+float_separator = "."
+
+# Thousads separator
+thousands_separator = ","
```

Comment:
Could you tell me the good value to put here @Moonbase59 :pray:?

Reply:
Both correct!

Reply:
Great!